frontend/public/components/storage-class-form.tsx

```diff
@@ -1006,6 +1006,7 @@ class StorageClassFormWithTranslation extends React.Component<
               id="storage-class-name"
               onChange={(event) => this.setStorageHandler('name', event.target.value.trim())}
               value={_.get(newStorageClass, 'name', '')}
+              data-test="storage-class-name-text"
```

Comment:
The form above has a data-test-id could we perhaps do something like
`cy.getById('storage-class-form').get('input#storage-class-name')`.